File: crates/cli/tests/fixtures/single-language/java-spring-boot-2/src/main/java/com/example/Application.java
package com.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.*;

import javax.servlet.http.HttpServletRequest;
import java.util.*;

@SpringBootApplication
@RestController
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

    @GetMapping("/")
    public Map<String, String> index(HttpServletRequest request) {
        return Map.of("message", "Spring Boot 2 Application",
                       "path", request.getRequestURI());
    }

    @GetMapping("/users")
    public List<Map<String, String>> getUsers() {
        return List.of(
                Map.of("name", "Alice"),
                Map.of("name", "Bob")
        );
    }
}

File: crates/cli/tests/fixtures/single-language/kotlin-gradle/src/main/kotlin/com/example/Application.kt
package com.example

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
@RestController
open class Application {
    @GetMapping("/")
    fun index(): String {
        return """{"message":"Kotlin App","version":"1.0.0","endpoints":["/","/health"]}"""
    }

    @GetMapping("/health")
    fun health(): String {
        return """{"status":"healthy"}"""
    }
}

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
